packages/core/types/breadcrumb.d.ts

```diff
@@ -1,9 +1,8 @@
 declare class Breadcrumb {
-  public name: string;
-  public metaData: object;
+  public message: string;
+  public metadata: object;
   public type: string;
   public timestamp: string;
-  constructor(name: string, metaData?: object, type?: string, timestamp?: string);
```

Comment:
Breadcrumb construction is not public, it can only be done via `leaveBreadcrumb()`.